website/content/docs/fork-workflow.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: Enable External Contributors with the Fork Workflow
+title: Fork Workflow
```

Comment:
I think a long title like this would be great for an announcement blog post, but gets awkward in the docs sidebar. I considered both "Fork Workflow" and "External Contributors", but the latter could be confused with contributors to the CMS project. I suppose "Enabling External Contributors" could work. Up to you. :)